client/src/components/NearestHospital/HospitalSidebar.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import styled from "styled-components";
+
+import { SearchBar } from 'antd-mobile';
+import { NavLink } from "react-router-dom";
+import LocalEmergencyNo from './LocalEmergencyNo';
+
+
+const HospitalSidebar = props => {
+
+const SearchBarContainer = styled.div`
+   padding: 5rem 0;
+`;
+
+const NavBar = styled.div`
+   border-top: 0.2px solid rgba(0, 0, 0, 0.5);
+   border-bottom: 0.2px solid rgba(0, 0, 0, 0.5);
+   ul {
+     list-style-type: none;
+     padding: 1rem 0;
+     li {
+        margin: 2rem 0;
+           a {
+             text-decoration: none;
+             transition: all .4s;
+             color: inherit;
+             padding-left: 2.5rem;
+           }
+           a:hover {
+             font-weight: bold;
+           }
+       }
+     }
+
+   }
+`;
+
+
+const StyledSearchBar = styled(SearchBar)`
+  &.am-search {
+    background-color: #fff;
+    border: 0.1rem solid rgba(0, 0, 0, 0.5);
+    border-radius: 4rem;
+  }
+  .am-search-cancel-show {
+    display: none;
+   }
+`;
+
+const ActiveLinkStyles = {
+  fontWeight: "bold",
+  borderLeft: "4px solid #425af2",
+  paddingLeft: "2rem"
+}
+
+
+  const [searchValue, setSearchValue] = useState('');
+
+  const onSearchInputChange = (value) => {
+    setSearchValue(value);
+    console.log(searchValue);
+
+  }
+  const clearSearch = () => {
+    setSearchValue('');
+  }
+
+
+
+return (
+   <div>
+     <SearchBarContainer>
+          <StyledSearchBar
+            value={searchValue}
+            placeholder=""
+            onClear={clearSearch}
+            onChange={onSearchInputChange}
+           />
+     </SearchBarContainer>
+     <NavBar>
+        <ul>
+           <li><NavLink activeStyle={ActiveLinkStyles} to="/nearest-hospital">Health Facilities</NavLink></li>
```

Comment:
On mobile for "Health Facilities" the layout should look like

![Screen Shot 2020-05-12 at 12 11 41 AM](https://user-images.githubusercontent.com/42228386/81649518-45704180-93e5-11ea-8db6-2a8a89400958.png)


currently:
![Screen Shot 2020-05-12 at 12 13 02 AM](https://user-images.githubusercontent.com/42228386/81649564-5b7e0200-93e5-11ea-88e2-dc4d22be427e.png)

Reply:
can i display a two pages in one? i had created a new route for confirmed-cases thats why i couldn't render it like that on mobile but maybe the confirmed cases should be a component. the desktop design made me think it should be another route.

Reply:
Ah I see.  But I think we can make this work without having `ConfirmedCases` be a separate pages?  

Under the "Health Facilities" tab we can just add the "ConfirmedCases" component to also show above "Your nearest health facilities".

Reply:
Yea let me try that